.changeset/light-tables-drive.md

```diff
@@ -0,0 +1,5 @@
+---
+"@latticexyz/world-modules": patch
+---
+
+Fixed `ERC20Module` to register the `TotalSupply` table when creating a new token.
```

Comment:
can we put some instructions for existing worlds/tokens to apply this fix as well?

Reply:
Added an example script!